spec/abci++/abci++_methods_002_draft.md

```diff
@@ -530,12 +536,13 @@ In the cases when _p_'s Tendermint is to broadcast `precommit nil` messages (eit
 
 * **Request**:
 
-    | Name      | Type  | Description                                                                              | Field Number |
-    |-----------|-------|------------------------------------------------------------------------------------------|--------------|
-    | extension | bytes | Sender Application's vote extension to be validated                                      | 1            |
-    | hash      | bytes | The header hash of the propsed block that the vote extension refers to                   | 2            |
-    | address   | bytes | [Address](../core/data_structures.md#address) of the validator that signed the extension | 3            |
-    | height    | int64 | Height of the block  (for sanity check).                                                 | 4            |
+    | Name              | Type  | Description                                                                              | Field Number |
+    |-------------------|-------|------------------------------------------------------------------------------------------|--------------|
+    | app_signed        | bytes | Optional information signed by the Application (not by Tendermint).                      | 1            |
+    | tendermint_signed | bytes | Optional information signed by Tendermint.                                               | 2            |
+    | hash              | bytes | The header hash of the propsed block that the vote extension refers to.                  | 3            |
+    | validator_address | bytes | [Address](../core/data_structures.md#address) of the validator that signed the extension | 4            |
+    | height            | int64 | Height of the block  (for sanity check).                                                 | 5            |
```

Comment:
Perhaps a different PR, but should we remove this as well given that we're not planning to implement it this release cycle?

Reply:
I'd prefer to keep the spec complete, so everyone can see what we are planning to do and comment on that.

Then, when the v0.36 (throttle) branch is created, we can remove the vote extension information from that version of the spec.

Adding and issue so we don't forget